charts/delphai-deployment/values.yaml

```diff
@@ -144,19 +144,24 @@ readinessProbe:
 ingress:
   enabled: false
 
-  # Define subdomain (regex) for HTTP (Helm template)
+  # Define subdomain (regex) for HTTP (evaluated as a Helm template)
   subdomain: 'app{{ if eq $.clusterValues.environment "review" }}.*{{ end }}'
-  # Define pefix for HTTP (Helm template)
+  # Define a prefix for HTTP (evaluated as a Helm template)
   urlPrefix: "/service/{{ $.Release.Name }}/"
 
+  # Enable exposing public client-facing API
+  publicApiEnabled: false
+  # Define a prefix (regex) for the public client-facing API
+  publicApiPrefix: null
+
   # Enable OAuth Proxy for HTTP
   oAuth: false
   oAuthRequireOneOfRoles: []
 
   # Publish GRPC
   grpc: false
 
-  # Run Cloudflare purge_cache after deployment
+  # Run Cloudflare purge_cache on deployment
```

Comment:
Hm, why?

It is called exactly after the helm deployment
> `"helm.sh/hook": "post-install,post-upgrade"`

Reply:
Reverted in 1fa4dc5. Copy-editing comments was out-of-scope of the current task.